src/components/DateRangePicker.jsx

```diff
@@ -584,14 +587,15 @@ class DateRangePicker extends React.PureComponent {
           block && styles.DateRangePicker__block,
         )}
       >
-        {enableOutsideClick && (
-          <OutsideClickHandler onOutsideClick={this.onOutsideClick}>
-            {input}
-            {this.maybeRenderDayPickerWithPortal()}
-          </OutsideClickHandler>
-        )}
-        {!enableOutsideClick && input}
-        {!enableOutsideClick && this.maybeRenderDayPickerWithPortal()}
+        {
+          enableOutsideClick
```

Comment:
i think the "two statements" form is a bit more readable than the ternary form, but i don't feel strongly about it

Reply:
I agree with this. Are you cool w/ me changing it, @TaeKimJR?